.github/dependabot.yml

```diff
@@ -16,5 +18,8 @@ updates:
     directory: "/utils/benchmarking/eventing"
     schedule:
       interval: "weekly"
-
-# Use https://dependabot.com/docs/config-file/validator/ to check for errors.
+updates:
+  - package-ecosystem: "gomod"
+    directory: "/utils/tracing/go"
+    schedule:
+      interval: "weekly"
```

Comment:
please add deployer, invoker and also function-images/tests/chained-function-eventing